packages/web-app-files/src/composables/viewMode/constants.ts

```diff
@@ -0,0 +1,4 @@
+export abstract class ViewModeConstants {
+  static readonly viewModeDefault: string = 'resource-table'
+  static readonly viewModeQueryName: string = 'view-mode'
+}
```

Comment:
Not a huge fan of this use class as enum pattern, I'd rather use typescript types for this nowadays

Reply:
I've followed what I found in another composable basically, idk if refactoring my/the other solution independently is within the scope of this PR?

Reply:
I agree with both of you 😆 Would like to get rid of it, but also fine in a separate PR.